deploy/charts/harvester/values.yaml

```diff
@@ -447,6 +447,8 @@ longhorn:
   ## defaults to "false", this will be enabled in ISO mode installation.
   enabled: false
 
+  createNamespace: true
```

Comment:
Will it succeed if this is enabled when the namespace exists?

Reply:
nope, but since the longhorn has not been installed so we would just assume the `longhorn-system` namespace doe not exist.